main.tf

```diff
@@ -274,7 +276,7 @@ resource "ibm_resource_instance" "governance_instance" {
 
   lifecycle {
     precondition {
-      condition     = contains(["eu-de", "us-south"], var.location)
+      condition     = contains(["eu-de", "us-south"], var.region)
```

Comment:
This is wrong, `watsonx.governance` is supported in various regions - 

<img width="737" height="615" alt="Image" src="https://github.com/user-attachments/assets/834b41d7-2df2-44ad-95d8-3cbe6ec2ff60" />